### DISCUSSION TITLE
How to input multiple file paths?

### DISCUSSION THREAD

**Discussion by danielkrajnik** [BurntSushi/ripgrep#2164] [General] [answered]
I use Everything to search file system  by name and ripgrep would be great to narrow down these results further by contents. I hoped that it will be possible to just copy-paste file paths (in quotes, delimited by new line) as arguments to ripgrep, but it doesn't seem to work.

Edit: solved - instead of delimiting them by newline, it worked when delimited by space. Duh, the part "as arguments" finally made me realize it. Thanks, sometimes just writing down the problem lets you get it.

**Comment by BurntSushi** [comment, marked as answer]
Answers have to be written as separate comments:

> Edit: solved - instead of delimiting them by newline, it worked when delimited by space. Duh, the part "as arguments" finally made me realize it. Thanks, sometimes just writing down the problem lets you get it.
